internal/backend/btree/cursor.go

```diff
@@ -0,0 +1,17 @@
+package btree
+
+type Cursor struct {
+	Node
+}
+
+func (c *Cursor) Search(key uint32) (interface{}, error) {
```

Comment:
nitpick, but maybe you should just throw a "not implemented" error or something until you implement the functions?

Reply:
👍 definitely agree!